odoo-shippable/scripts/entrypoint_image

```diff
@@ -71,7 +71,7 @@ def docker_entrypoint():
         subprocess.call(cmd, shell=True)
 
     # Start postgresql service
-    cmd = 'sudo /etc/init.d/postgresql start ' + os.environ.get('PSQL_VERSION', '')
+    cmd = '/etc/init.d/postgresql start ' + os.environ.get('PSQL_VERSION', '')
```

Comment:
@moylop260  If you only need the sudo for postgres service. Then if you can eliminate the sudoers